packages/uicore/src/components/InputWithIdentifier/InputWithIdentifier.tsx

```diff
@@ -45,7 +45,7 @@ export interface InputWithIdentifierProps {
    */
   isIdentifierEditable?: boolean
   /**
-   * @default 63
+   * @default 128
```

Comment:
Should this be 127 instead of 128? Because earlier it was 63 instead of 64.

Reply:
@mayankVermaHarness : I had discussed same here : https://harness.atlassian.net/browse/PL-30690?focusedCommentId=399042